source/intro.tex

```diff
@@ -345,8 +343,8 @@
 
 \definition{iostream class templates}{defns.iostream.templates}
 \defncontext{library}
-templates, defined in \ref{input.output},
-that take two template arguments
+templates that take two template arguments
+and are declared in header \libheader{iosfwd}
```

Comment:
Can we keep the two conditions the other way around? It makes more sense to me to start with all templates in `<iosfwd>` and then filter by "takes two arguments" than to start with the universe of all templates taking two arguments and then filter by "declared in `<iosfwd>`".

Reply:
Fixed.